kinematics.go

```diff
@@ -128,12 +113,11 @@ var MaxInverseKinematicIteration int = 50
 // InverseKinematics calculates joint angles to achieve an XyzWxyz end effector
 // position given the desired XyzWxyz coordinates and the robotic arm
```

Comment:
Is it worth renaming `XyzWxyz` to something more conventional like `Pose`?

Reply:
Yes, we could!